Use BLST portable feature to enable runtime ADX instruction set check (#1630)
input-output-hk/mithril: CHANGELOG.md

- Chain observers support the retrieval of the current Cardano chain point.

> - Deprecate `portable` feature of `mithril-stm` and `mithril-client`, BLST runtime check of intel ADX instruction set

Reviewer: Maybe you could also say its has been removed from `mithril-common` and other crates?

>  is now enabled. The portable feature now do nothing and should be removed from your crate dependencies.

Reviewer: ```suggestion
 is now enabled. The portable feature now has no effect and should be removed from crate dependencies.
```
